kep/219-pod-allocatable-ip-scheduling/README.md

```diff
@@ -0,0 +1,49 @@
+# Pod Allocatable IP<!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc -->
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+- [Use Cases](#use-cases)
+- [Design Details](#design-details)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Summary
+
+This plugin takes node's `.spec.PodCIDR` into account in scheduling cycle. When the density of pods in a node is relatively high, nodes might not have sufficient allocatable Pod IP to allocate.
+Even though kubelet has its own `--max-pods` option flag to limit the total number of Pods on nodes, we can filter those nodes who have insufficient Pod IP to allocate in scheduling cycle. 
+
+## Motivation
+[kube-controller-manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/) provides the `--allocate-node-cidrs` options to mark Pod CIDR in each node's `.spec.PodCIDR` field.
+When a node has already allocated all the Pod IPs from `.spec.PodCIDR`, a scheduling Pod will ignore this node in scheduling cycle. It avoids the situation that pods are binding to insufficient Pod IP nodes,
+which is useful when a Pod cannot be scheduled due to the limited Pod IP allocation constraint in one node. This constraint is likely to result in Pods stucking in `ContainerCreating` phase.
```

Comment:
How about "Without it, a pod can be scheduled and then stuck in `ContainerCreating` phase."